src/NServiceBus.Core/obsoletes.cs

```diff
@@ -1055,4 +1055,30 @@ public partial interface IInvokeHandlerContext
     }
 }
 
+namespace NServiceBus
+{
+    using Features;
+
+    // Just to make sure we remove it in v8. We keep it around for now just in case some external feature 
```

Comment:
What if someone disables the feature using the fullname?

I guess the core would no longer work and never did right? So this is safe I think

Reply:
There is no public api to disable using a string

https://github.com/Particular/NServiceBus/blob/master/src/NServiceBus.Core/Features/EndpointConfigurationExtensions.cs

so since a `Type` is always needed users would never be able to do this?